arkouda_distance_wserver/client/arkouda_distance/distance.py

```diff
@@ -0,0 +1,26 @@
+from arkouda import pdarray
+from arkouda.client import generic_msg
+from arkouda.pdarrayclass import parse_single_value
+from typing import cast
+import math
+
+def dot(u: pdarray, v: pdarray):
```

Comment:
should we include `@typechecked` from the typeguard package?

Reply:
I would think so. I will add that as well before pushing up